### PR TITLE
[RELEASE-1.18] SRVKE-934: skip HPA

### PR DESCRIPTION
**Pull request by matzew**
Still relevant for 1.18 release

/assign @maschmid

### CODE REVIEW

File: openshift-knative-operator/hack/update-manifests.sh [hunk 1]
```diff
@@ -110,6 +110,9 @@ git apply "$root/openshift-knative-operator/hack/001-eventing-namespace-deletion
 # TODO: propose to upstream
 git apply "$root/openshift-knative-operator/hack/002-openshift-eventing-role.patch"
 
+# For SRVKE-914 we disable HPA:
```
Comment (maschmid) [inline openshift-knative-operator/hack/update-manifests.sh:113]:
```suggestion
# For SRVKE-934 we disable HPA:
```

**Comment by matzew** [comment]
```
level=error msg=  42: resource "aws_route53_record" "api_external_alias" {
```

/retest

**Pull request merged**